### PR TITLE
AzureSQL UC commit

### PR DESCRIPTION
**Pull request by ldmahale108**
Signed-off-by: Laxmikant Mahale <Laxmikant.Mahale@ibm.com>

### CODE REVIEW

File: docs/available_plugins.md [hunk 1]
```diff
@@ -20,4 +20,5 @@
...
-|            [DocumentDB](../filter-plugin/logstash-filter-documentdb-aws-guardium/README.md)            | 4.0        | AWS                 | IBM                  | CloudWatch (pull)                    |                                                                   [GDP](../filter-plugin/logstash-filter-documentdb-aws-guardium/DocumentDBOverCloudwatchPackage)             
\ No newline at end of file
+|            [DocumentDB](../filter-plugin/logstash-filter-documentdb-aws-guardium/README.md)            | 4.0        | AWS                 | IBM                  | CloudWatch (pull)                    |                                                                   [GDP](../filter-plugin/logstash-filter-documentdb-aws-guardium/DocumentDBOverCloudwatchPackage)             
+|            [AzureSQL](../filter-plugin/logstash-filter-azure-sql-guardium/README.md)            | 12.0.2000.8        | Cloud                 | IBM                  | JDBC (pull)                    |                                                                   [GDP](../filter-plugin/logstash-filter-azure-sql-guardium/AzureSQLOverJdbcPackage)
```
Comment (ofer-haim) [inline docs/available_plugins.md:24]: Is there a version of Azure SQL in other Cloud services? If not, I think it's best to write Azure instead of Cloud

**Review by ofer-haim — APPROVED** [review]
Looks good. Please make sure this is the exact same content the was reviewed and tested in the internal repo

File: docs/available_plugins.md (hunk 1 above)
Comment (ldmahale108) [reply docs/available_plugins.md]: Done.

File: filter-plugin/logstash-filter-azure-sql-guardium/README.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    6. Update the input section to add the details from azureSQL.conf file's input part, omitting the keyword "input{" at the beginning and its corresponding "}" at the end.
+	Note : • For Guardium Data Protection version 11.3, add the following line to the input section:
+	'jdbc_driver_library => "${THIRD_PARTY_PATH}/mssql-jdbc-7.4.1.jre8.jar"'
+		• If auditing is configured way long back and UC is configured at later point of time, still UC will process all the previous older records as well till date, since it is already audited by the DB.
+		• "type" field should match in input and filter configuration section. This field should be unique for every individual connector added.
+		• If using two jdbc plugin on same machine ,last_run_metadata_path file name should be different.
+    7. Update the filter section to add the details from azureSQL.conf file's filter part, omitting the keyword "filter{" at the beginning and its corresponding "}" at the end.
+    8. Click Save. Guardium validates the new connector, and enables the universal connector if it was disabled. After it is validated, the connector appears in the Configure Universal Connector page.
+
```
Comment (PratikshaSonawane) [inline filter-plugin/logstash-filter-azure-sql-guardium/README.md:115]: Please add pagination related information, which helps avert the MemoryIndexOutOfBound exception, as it is added for other JDBC plugins.

**Review by PratikshaSonawane — CHANGES_REQUESTED** [review]

File: filter-plugin/logstash-filter-azure-sql-guardium/README.md (hunk 2 above)
Comment (ldmahale108) [reply filter-plugin/logstash-filter-azure-sql-guardium/README.md:119]: Done.

**Review by PratikshaSonawane — APPROVED** [review]

**Pull request merged**